CMakeLists.txt

```diff
@@ -266,3 +269,10 @@ endif()
 if(IPC_TOOLKIT_TOPLEVEL_PROJECT AND IPC_TOOLKIT_BUILD_PYTHON)
   add_subdirectory(python)
 endif()
+
+# Inform users about configuration choices
+message(STATUS "Building IPC Toolkit with the following options:")
+message(STATUS "  - CUDA support: ${IPC_TOOLKIT_WITH_CUDA}")
+message(STATUS "  - SIMD support: ${IPC_TOOLKIT_WITH_SIMD}")
+message(STATUS "  - Rational intersection: ${IPC_TOOLKIT_WITH_RATIONAL_INTERSECTION}")
+message(STATUS "  - Robin map support: ${IPC_TOOLKIT_WITH_ROBIN_MAP}")
```

Comment:
These are useful messages, but I do not want to pollute the CMake log. Can you mark them as `DEBUG` instead of `STATUS`?